sklearn/linear_model/ridge.py

```diff
@@ -1041,9 +1041,9 @@ class _RidgeGCV(LinearModel):
     We want to solve (K + alpha*Id)c = y,
     where K = X X^T is the kernel matrix.
 
-    Let G = (K + alpha*Id)^-1.
+    Let G = (K + alpha*Id).
 
-    Dual solution: c = Gy
+    Dual solution: c = G^(-1)y
```

Comment:
indeed the existing code was using the notation G to denote the inverse of the matrix G in the reference